reflectable/lib/reflectable_builder.dart

```diff
@@ -6,19 +6,30 @@ library reflectable.src.reflectable_builder;
 
 import 'dart:async';
 import 'package:barback/src/transformer/barback_settings.dart';
+import 'package:build/build.dart';
+import 'package:build_config/build_config.dart';
 import 'package:build_barback/build_barback.dart';
 import 'package:build_runner/build_runner.dart';
 import 'package:reflectable/transformer.dart';
 
+TransformerBuilder reflectableBuilder(BuilderOptions options) {
+  var config = new Map<String, Object>.from(options.config);
```

Comment:
Just as an fyi, this corresponds to the `options` field in a custom build.yaml file, if a package has one. So for instance they could set up:

```yaml
targets:
  my_package:
    builders:
      reflectable:
        options:
          formatted: true
```